doc/how_to_guides/isaac_panda/launch/python.sh

```diff
@@ -18,6 +18,6 @@ do
     NEW_ARGS="${NEW_ARGS} ${CUR_SCRIPT_DIR}/${arg}"
 done
 
-pushd ${ISAAC_SCRIPT_DIR}
+pushd ${ISAAC_SCRIPT_DIRS[${#ISAAC_SCRIPT_DIRS[@]}-1]}
```

Comment:
This does mean that someone will have to manually check that ALL released versions of Isaac, now and into the future, are compatible with this tutorials omnigraph creation script. Pretty sure this is safe right now but Nvidia has been known to make breaking changes to that API. What do you think about adding some checks to verify the version against a known working list?

Reply:
Hmm, your suggestion is right.
I don't trust nvidia either.
I checked this working in `2022.2.1`, so we should check if the package is `2022.2.1` or `2022.2.0`.

Reply:
I have also run this back to `1.1` but would have to test it again to make sure it still works. I'm okay with this launch script only supporting `2.0` and `2.1` for now.

Reply:
Thank you for your information. I updated.